src/analytics.js

```diff
@@ -56,3 +58,30 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {!Element} element
+ * @param {!JSONType} config
+ * @param {Document=} parentDoc
+ */
+export function insertAnalyticsElement(element, config, parentDoc) {
```

Comment:
I worked on an experimental PR yesterday, and found that to `#tirggerAnalyticsEventForTarget` I will need to trigger at the corresponding analytics element. Do you think it's reasonable for me to make this function return a promise?